flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -635,6 +636,35 @@ public void testTemporaryFunction() throws Exception {
         tableEnv.executeSql("drop temporary function if exists foo");
     }
 
+    @Test
+    public void testTemporaryFunctionUDAF() throws Exception {
+        // create temp function
+        tableEnv.executeSql(
+                String.format(
+                        "create temporary function temp_count as '%s'",
```

Comment:
Could you like to test the udtf you define in this PR?

Reply:
done